src/main/java/net/dv8tion/jda/internal/entities/UserImpl.java

```diff
@@ -155,6 +158,18 @@ public boolean isFake()
         return fake;
     }
 
+    @Nonnull
+    @Override
+    public EnumSet<UserFlag> getFlags()
+    {
+        return flagsSet;
+    }
+    
+    public int getFlagsRaw()
```

Comment:
Add this to the interface. This can be useful for better compatibilty when discord adds new flags.

Reply:
Added